keystrokes/src/components/utils/LinkButton.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Button from '@material-ui/core/Button';
+import { withRouter } from 'react-router'
+
+const LinkButton = (props) => {
```

Comment:
the indentation of this file looks odd compared to others. Spaces vs tabs?

Reply:
yea i copied this from another project i did, i realize i dont actually need this, so im going to just remove it from the project.